components/analytics/doughnut_chart.tsx

```diff
@@ -4,7 +4,8 @@
 import deepEqual from 'fast-deep-equal';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
-import Chart, {ChartData} from 'chart.js';
+import Chart from 'chart.js/auto';
```

Comment:
wondering 🤔 why we need `/auto`

Reply:
The new version require either registering via:

```
Chart.register(...registerables); // registerables are imported from chart.js as well
```

and auto does that (importing and registering the relevant controllers / line/doughnut etc).